h/accounts/schemas.py

```diff
@@ -185,12 +185,16 @@ class RegisterSchema(CSRFSchema):
             colander.Length(
                 min=models.USERNAME_MIN_LENGTH,
                 max=models.USERNAME_MAX_LENGTH),
-            colander.Regex('(?i)^[A-Z0-9._]+$'),
+            colander.Regex(
+                '(?i)^[A-Z0-9._]+$',
+                msg=_("Must contain only letters, numbers, periods and "
```

Comment:
Hmm, I'm leading towards including it because Wikipedia says most US style guides include it, including Strunk & White which I happen to have here